src/About/SupportCard.tsx

```diff
@@ -52,6 +70,39 @@ export default () => {
                     label="Create system report"
                 />
             </Section>
+            <Section title="Verbose Logging">
+                <p>
+                    Aid our support team with additional log information. Enable
+                    this only when necessary as the log will grow quickly.
+                </p>
+                <Toggle
+                    id="enableVerboseLoggin"
+                    label="VERBOSE LOGGING"
+                    onToggle={() => {
+                        if (!verboseLogging)
```

Comment:
Hmm, maybe I'm just brainfarting here, but shouldn't this conditional be switched around? If `verboseLogging` is `true` we want to set TRACE-level logging right?

Reply:
The condition gets toggled after after the if-else statement.

Reply:
Can we swap the order of events then? Otherwise it's confusing, at least to me.